openconnect_sso/authenticator.py

```diff
@@ -173,6 +181,9 @@ class AuthCompleteResponse:
     session_token = attr.ib(converter=str)
     server_cert_hash = attr.ib(converter=str)
 
+    def asdict(self):
+        return self.__dict__
```

Comment:
I think I should replace this with `attr.asdict(self)`
https://www.attrs.org/en/stable/api.html#attr.asdict